lessons/Early-Math/associative-property/langs/en-us/pages/true-or-false.adoc

```diff
@@ -0,0 +1,22 @@
+= True or False? (Associative Property)
+
+++++
+<style>
+div.circleevalsexp { width: auto; }
+td > .content > .paragraph > * { vertical-align: middle; }
+</style>
+++++
+
+Is the equation represented by the two Circles true or false? Explain why, and then describe what you would change to make the _false_ equations _true_.
+
+[.FillVerticalSpace,cols="^.^1a,^.^8a,^.^5a", stripes="none", options="header"]
+|===
+|	 | Circles.																	   |
+True or False? Explain.
+| @n |@show{(coe '(* (* 8 9) 10))}	@math{=} @show{(coe '(* (* 8 9) 10))}  |
```

Comment:
confirming you meant for the first 2 to be exactly the same. I see pros and cons to this - not objecting. just making sure it was intentional